db/migrate/20201103143405_fix_google_cloud_volume_sti.rb

```diff
@@ -0,0 +1,30 @@
+class FixGoogleCloudVolumeSti < ActiveRecord::Migration[5.2]
+  class ExtManagementSystem < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.inheritance_column = :_type_disabled
+  end
+
+  class CloudVolume < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.inheritance_column = :_type_disabled
+
+    belongs_to :ext_management_system,
+               :foreign_key => :ems_id,
+               :class_name  => "FixGoogleCloudVolumeSti::ExtManagementSystem"
+  end
+
+  GOOGLE_CLOUD_KLASS        = "ManageIQ::Providers::Google::CloudManager".freeze
+  GOOGLE_CLOUD_VOLUME_KLASS = "ManageIQ::Providers::Google::CloudManager::CloudVolume".freeze
+
+  def up
+    gce_relation = ExtManagementSystem.in_my_region.where(:type => GOOGLE_CLOUD_KLASS)
```

Comment:
I totally forgot before merging, but can you do a follow up PR with a `say_with_time` around all these?

Reply:
:+1: https://github.com/ManageIQ/manageiq-schema/pull/534